src/main/java/io/vertx/spi/cluster/zookeeper/ZookeeperClusterManager.java

```diff
@@ -37,7 +41,7 @@
  *
  * @author Stream.Liu
  */
-public class ZookeeperClusterManager implements ClusterManager, PathChildrenCacheListener, ConnectionStateListener {
+public class ZookeeperClusterManager implements ExtendedClusterManager, PathChildrenCacheListener, ConnectionStateListener {
```

Comment:
Is extended cluster manager required ? 
It was initially made to work around a Hazelcast limitation